7.15 (#537)
enonic/doc-xp: docs/release/upgrade.adoc

After upgrading XP, check for new versions of your installed applications from Enonic market.
We generally always recommend updating your apps to the latest version!

== v7.15 notes

* The image service now generates progressive JPEG images by default. This can be changed back in <<deployment/config#image>> by setting `progressiveJpeg` to empty string.

* GZIP HTTP compression is disabled by default. It can be enabled in <<deployment/config#jetty, Jetty config>>. By setting `gzip.enabled = true`.

> * When content is created `image/webp` and `image/avif` media types create `media:image` content type, instead of `media:unknown`. Old versions of Content Studio might not work correctly with WebP and AVIF media types. Upgrae t the latest version of Content Studio.

Reviewer: ```suggestion
* `image/webp` and `image/avif` media types will now produce content of `media:image` type, instead of `media:unknown`. Old versions of Content Studio might not work correctly with WebP and AVIF media types. Upgrade to the latest version of Content Studio.
```

* <<deployment/config#repo, Repo config>> `cache.size` parameter no longer works. Use `cache.capacity` instead.

* Vacuuming no longer clears unused blobs from deleted repositories. <<deployment/backup-restore#vacuum, Vacuuming>> documentation has been updated to reflect this change.

* Vacuuming now deletes old snapshots by default. <<deployment/backup-restore#vacuum, Vacuuming>> documentation has been updated to reflect this change.

> * Andy vacuuming that affects blobs will delete snapshots after the vacuuming is done. <<deployment/backup-restore#vacuum, Vacuuming>> documentation has been updated to reflect this change.

Reviewer: ```suggestion
* Vacuuming that affects blobs will delete snapshots once completed. <<deployment/backup-restore#vacuum, Vacuuming>> documentation has been updated to reflect this change.
```
